customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/persistence/JpaCertificateRepository.java

```diff
@@ -51,6 +51,23 @@ public List<Certificate> get(Termination termination) {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public void remove(Termination termination) {
+    final var terminationEntity = terminationEntityRepository
+        .findByTerminationId(termination.terminationId().id());
+
+    if (terminationEntity.isEmpty()) {
+      return;
```

Comment:
Maybe we should throw a NotFoundException if we expect to find a termination to remove?

Reply:
You have a good point. It is better to do as you suggest as it is correct that we do expect a termination to exist. 

Will fix!

Reply:
Fixed!